lib/plugins.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+var _ = require('lodash');
+var prefix = 'gemini-';
+
+module.exports = {
+    load: function(gemini, options) {
+        var enabledPlugins = _.reject(_.pairs(options.plugins), function(p) {
+            return !p[1];
+        });
+
+        _.forEach(enabledPlugins, function(plugin) {
+            var shortName = plugin[0].replace(prefix, '');
+            var packageName = prefix + shortName;
```

Comment:
Maybe, it will be clearer if rewrite it as `packageName = _.startsWith(plugin[0], prefix)? plugin[0]: prefix + plugin[0]`